antsibull/data/docsite/role.rst.j2

```diff
@@ -0,0 +1,286 @@
+.. Document meta
+
+:orphan:
+
+{# If we can put together source and github repo, we could make the Edit me of github button work.
```

Comment:
This would be great, but not all collections have code hosted on GitHub - can we build this functionality so that the button only appears on pages where we know the collection repo and we know that collection repo is on GitHub?

Reply:
This is basically a copy'n'paste from `plugin.rst.j2` - I have no intention on doing anything with this right now :) If this is ever implemented, it should be implemented for both templates simultaneously.

Reply:
Ah, understood.